chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/CertificationResult.java

```diff
@@ -100,6 +100,9 @@
 
     @XmlTransient
     private List<MacraMeasure> allowedMacraMeasures;
+    
+    @XmlTransient
+    private List<TestFunctionality> allowedTestFunctionalities;
```

Comment:
Will this addition to this file change the XML download file? If so, please add the XSD-MODIFICATION file to that UI project. Would also recommended verifying that our XSD diff step in the DEV plan works correctly when you do build this project